src/css/components/menu/_menu-popup.scss

```diff
@@ -9,6 +9,11 @@
   border-top-color: rgba($primary-background-color, $primary-background-transparency); // Same as ul background
 }
 
+.vjs-pip-window .vjs-menu-button-popup .vjs-menu {
+  left: unset;
+  right: 1em; // Extra offset for last menu button in pip window, as fullscreen button not present
+}
```

Comment:
Consider this temporary. This needs a better solution longer term that only modifies the last _visible_ menu, but this situation could apply in situations other than PiP so deserves tackling separately to thi PR.